lib/vmpooler/api/v1.rb

```diff
@@ -645,6 +645,9 @@ def invalid_template_or_path(payload)
           result[params[:hostname]]['state'] = 'destroyed'
         elsif rdata['checkout']
           result[params[:hostname]]['running'] = ((Time.now - Time.parse(rdata['checkout'])) / 60 / 60).round(2)
+          result[params[:hostname]]['remaining'] = ((Time.parse(rdata['checkout']) + rdata['lifetime'].to_i*60*60 - Time.now) / 60 / 60).round(2)
+          result[params[:hostname]]['start_time'] = Time.parse(rdata['checkout'])
```

Comment:
Personally, I'd like to see an explicit format chosen here, instead of at the whim of Ruby. i.e. is is UTC or does it include the originating timezone.

Reply:
I'd love to see RFC3339 here

Reply:
👍  to RFC3339

Reply:
Updated!